clients-interpreted.yml

```diff
@@ -150,6 +150,19 @@
       - project:
             level: 3
             feature: [beetmover]
+
+# special entry for translations, where the production repository is L1, and thus doesn't
+# match the block above
+- client:
+      project/releng/scriptworker/v2/beetmover/prod/firefoxci-{trust_domain}-1:
+          description: "{trust_domain} level 1 beetmover scriptworker"
+          scopes:
+              - queue:claim-work:scriptworker-k8s/{trust_domain}-1-beetmover
+              - queue:worker-id:{trust_domain}-1-beetmover/{trust_domain}-1-beetmover-*
+  for:
+      - project:
+            alias: [translations, staging-firefox-translations-training]
```

Comment:
should this be `{level: 1, feature: [beetmover]}`, instead of listing the projects?